### DISCUSSION TITLE
Is there any way to disable/reenable an existing alarm instead of deleting it?

### DISCUSSION THREAD

**Discussion by tehniemer** [leikoilja/ha-google-home#851] [Q&A]
I haven't been able to find an answer to this specific question anywhere. I want to be able to disable my kid's alarms on non-school days, then reenable it later, but it seems that my only option is to delete it.

**Comment by KapJI** [comment]
You can check if such functionality is available in Google Home app.

Alarms may have `INACTIVE` state but I think that's only possible to set via a voice command.

**Comment by tedr91** [comment]
I completely agree!  I would personally find it way more useful to have enable_alarm and disable_alarm services available (that set state to SET and INACTIVE respectively) than simply delete_alarm.
